scripts/core/ui/ui.ts

```diff
@@ -1048,7 +1048,7 @@ function validationDirective() {
             });
 
             scope.$watch(attrs.sdValidationError, (errors) => {
-                if (isEmpty(errors)) { // valid
+                if (errors == null || errors === '' || errors === false || (errors.length != null && errors.length === 0)) {
```

Comment:
How about `Array.isArray(errors) && errors.length > 0`? Or isn't this what you are trying to check?